services/slackServices/shoutout.js

```diff
@@ -6,6 +6,19 @@ const User = require('../../data/dbModels/users');
 const { clientUrl } = require('../../config');
 const moment = require('moment');
 
+function randomGifs() {
+  const gifArray = [
+    'https://media0.giphy.com/media/xJjs8eGVbjNYY/giphy.gif',
+    'http://giphygifs.s3.amazonaws.com/media/dsMFrxB2agKf6/giphy.gif',
+    'https://media.giphy.com/media/iyGfMZYxDQBSE/giphy.gif',
+    'https://media.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif',
+  ];
+
+  const randomInt = Math.floor(Math.random() * Math.floor(gifArray.length));
+  console.log(randomInt);
```

Comment:
Hey @ogwurujohnson, remove the console.log here

Reply:
done